src/service/uploads/handler.ts

```diff
@@ -433,7 +416,13 @@ function generateDefaultValue(dataType: string, columnType: string) {
   const isDateTime = dataType == 'datetime'
 
   if (isEnum) {
-    return 1
+    if (typeof columnDefault === 'string') {
+      // remove quote at the beginning and end of string
+      return columnDefault.replace(/^'|'$/g, '')
+    }
+    const match = columnType.match(/^enum\('(.+?)'/)
```

Comment:
Just return 1 will get you first value of enum, no need to parse enum(value1, value2), as stated in mariadb documentation here https://mariadb.com/kb/en/enum/

Reply:
ahh thanks, I have restored previous enum handling